src/EpicClipboard.ahk

```diff
@@ -4,119 +4,284 @@
-Gui Add, Button, x195 y235 w90 h50 gNo_RPG, No 👎
-Gui Add, Button, x15 y285 w90 h50 gFish_RPG, Fish 🐟 ;⚔
-Gui Add, Button, x105 y285 w90 h50 gChop_RPG, Chop 🎄 ;🎃
-Gui Add, Button, x195 y285 w90 h50 gCatch_RPG, Catch 🎺
-
+IniRead, lootbox_str, %A_ScriptDir%\config.ini, Commands, lootbox
+IniRead, enchant_str, %A_ScriptDir%\config.ini, Commands, enchant
+IniRead, farm_str, %A_ScriptDir%\config.ini, Commands, farm
+
+;GUI Settings
+Gui, Font, s10, Roboto
+
+; Hero Panel
+Gui Add, GroupBox, x10 y5 w280 h65, Hero
+Gui Add, Button, x15 y25 w90 h40 gProfile_RPG, Stats 🧑
+Gui Add, Button, x105 y25 w90 h40 gItems_RPG, Items 🎾
+Gui Add, Button, x195 y25 w90 h40 gHeal_RPG, Heal 🏥
+
+; Actions Panel
+Gui Add, GroupBox, x10 y70 w280 h145, Actions
+Gui Add, Button, x15 y90 w90 h40 gHunt_RPG, Hunt 🏹
+Gui Add, Button, x105 y90 w90 h40 gAdv_RPG, Adv 🗡
+Gui Add, Button, x195 y90 w45 h40 gCd_RPG, CD ⏰
+Gui Add, Button, x240 y90 w45 h40 gRd_RPG, RD ⌛
+Gui Add, Button, x15 y130 w90 h40 gWork_RPG, Work ⛏
+Gui Add, Button, x105 y130 w90 h40 gTrain_RPG, Train 💪
+Gui Add, Button, x195 y130 w90 h40 gDuel_RPG, Duel ⚔️
+Gui Add, Button, x15 y170 w45 h40 gFarm_RPG, Farm 🚜
+Gui Add, Button, x60 y170 w45 h40 gFarmC_RPG, Farm 🌾
+Gui Add, Button, x105 y170 w90 h40 gQuest_RPG, Quest ❓
+Gui Add, Button, x195 y170 w90 h40 gEQuest_RPG, Epic Quest ❓
+
+; Reply Panel 
+Gui Add, GroupBox, x10 y215 w280 h105, Reply
+Gui Add, Button, x15 y235 w90 h40 gYes_RPG, Yes ✔️
+Gui Add, Button, x15 y275 w90 h40 gNo_RPG, No ❌
+Gui Add, Button, x105 y235 w60 h40 gA_RPG, A 🔠
+Gui Add, Button, x165 y235 w60 h40 gB_RPG, B 🔠
+Gui Add, Button, x225 y235 w60 h40 gC_RPG, C 🔠
+Gui Add, Button, x105 y275 w60 h40 g1_RPG, 1 🔢
+Gui Add, Button, x165 y275 w60 h40 g2_RPG, 2 🔢
+Gui Add, Button, x225 y275 w60 h40 g3_RPG, 3 🔢
+
+; Pings Panel 
+Gui Add, GroupBox, x10 y320 w280 h105, Pings
+Gui Add, Button, x15 y340 w90 h40 gSummon_RPG, Summon 📦
+Gui Add, Button, x105 y340 w90 h40 gTimeFight_RPG, Time To Fight 🛡️
+Gui Add, Button, x195 y340 w90 h40 gJoin_RPG, Join 🍪
+Gui Add, Button, x15 y380 w90 h40 gFish_RPG, Fish 🐟 ;⚔
+Gui Add, Button, x105 y380 w90 h40 gChop_RPG, Chop 🎄 ;🎃
+Gui Add, Button, x195 y380 w90 h40 gCatch_RPG, Catch 🎺
+
+; Rewards Panel
+Gui Add, GroupBox, x10 y425 w280 h105, Rewards
+Gui Add, Button, x15 y445 w135 h40 gLootbox_RPG, 📦 Lootbox 📦
+Gui Add, Button, x150 y445 w135 h40 gVote_RPG, 🗳️ Vote 🗳️
+Gui Add, Button, x15 y485 w135 h40 gDaily_RPG, 📅 Daily 📅
+Gui Add, Button, x150 y485 w135 h40 gWeekly_RPG, 📆 Weekly 📆
+
+; Enchant Panel
+Gui Add, GroupBox, x10 y530 w280 h65, Enchant
+Gui Add, Button, x15 y550 w135 h40 gESword_RPG, ✨ Sword ✨
+Gui Add, Button, x150 y550 w135 h40 gEArmor_RPG, ✨ Armor ✨
+
+; Window Settings
 Gui, +Alwaysontop -MaximizeBox
-Gui Show, w300 h345, EPIC Clipboard
+Gui Show, w295 h600, EPIC RPG Clipboard
 Return
-
+
+; Command Lines 
 Profile_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg p, ahk_exe Discord.exe
+    SendInput {enter}
```

Comment:
While I agree that having the ability to auto-send the command on click is a great QoL, it would remove the aspect of being a clipboard (like the one from Windows and smartphone keyboards) and make the tool banned from being used to play EPIC RPG. 

I would like to keep the public version as "legal" as possible so please remove every `SendInput {enter}` from the commands.

Tip: you can put `{Enter}` after string commands without the need of ` SendInput`, e.g:
> ControlSend,, rpg p {Enter}, ahk_exe Discord.exe

Reply:
Noted x3 on this, I realize your concerns with this and will remove it as requested.

Reply:
Also thank you for the tip.